dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsModalColumnHeaders.ts

```diff
@@ -5,12 +5,18 @@ export const euTaxonomyForNonFinancialsModalColumnHeaders = {
     share: "Share",
     revenue: "Revenue",
     revenuePercent: "Revenue (%)",
-    ClimateMitigation: "Climate change mitigation",
-    ClimateAdaptation: "Climate change adaptation",
-    Water: "Water and marine resources",
-    CircularEconomy: "Circular economy",
-    PollutionPrevention: "Pollution prevention",
-    Biodiversity: "Biodiversity and ecosystems",
+    substantialContributionToClimateChangeMitigation: "Climate Change Mitigation",
```

Comment:
Could this be simplified? Seems redundant to differentiate between substantialContribution and dnsh, if they translate to the same headers.

Reply:
Another point: Why is the thing not renamed here. E.g. `substantialContributionToClimateChangeMitigation` should be `substantialContributionToClimateChangeMitigationInPercent` right?
And why doesn't this break anything? Or does it?

Reply:
I am on it

Reply:
For the sake of consistency, I adapted these names to include the suffix `InPercentage`. However, this would not have been necessary in order to make the view page compile

Reply:
ok